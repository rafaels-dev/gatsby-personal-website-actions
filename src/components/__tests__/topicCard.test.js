import React from 'react';
import renderer from 'react-test-renderer';

import TopicCard from '../topicCard.js';

describe('TopicCard', () => {
    it('Renders correctly', () => {
        const topic = {
            description: 'My Description'
        };
        const tree = renderer
            .create(<TopicCard topic={topic} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
});