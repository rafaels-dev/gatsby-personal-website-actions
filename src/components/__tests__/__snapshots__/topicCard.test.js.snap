// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopicCard Renders correctly 1`] = `
<div
  className="github-component position-relative height-full text-center border border-gray-light rounded-1 bg-white p-5"
>
  <div
    className="bg-blue-light f4 text-gray-light text-bold rounded-1 flex-shrink-0 text-center mx-auto mb-3"
    style={
      Object {
        "height": "64px",
        "lineHeight": "64px",
        "width": "64px",
      }
    }
  >
    #
  </div>
  <p
    className="f3 lh-condensed text-center mb-0 mt-1"
  />
  <p
    className="f5 text-gray text-center mb-0 mt-1"
  >
    My Description
  </p>
</div>
`;
